src/main/java/Service/ProductService.java

```diff
@@ -0,0 +1,103 @@
+package Service;
+
+import DatabaseInteractin.DataBaseInteracting;
+import DatabaseInteractin.ExecuteQuerys;
+import DatabaseInteractin.Product;
+import generated.addProductRequest;
+import generated.addProductResponse;
+import generated.addShoppingCartRequest;
+import generated.addShoppingCartResponse;
+import generated.listByIdRequest;
+import generated.listByIdResponse;
+import generated.listProductRequest;
+import generated.listProductResponse;
+import generated.productGrpc;
+import io.grpc.stub.StreamObserver;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+public class ProductService extends productGrpc.productImplBase {
+  ExecuteQuerys executeQuerys = new ExecuteQuerys();
+  DataBaseInteracting dataBaseInteracting = new DataBaseInteracting();
+
+  public ProductService() throws SQLException {
+  }
+
+  @Override
+  public void listById(listByIdRequest request, StreamObserver<listByIdResponse> responseObserver) {
+    try {
+      int id = request.getId();
+      ResultSet resultSet = dataBaseInteracting.getProductsById(id);
+      while (resultSet.next()) {
+        String name = resultSet.getString(2);
+        int stock = resultSet.getInt(3);
+        float price = resultSet.getFloat(4);
+        listByIdResponse listByIdResponse = generated.listByIdResponse.newBuilder().
```

Comment:
You can omit the package name

Reply:
Searching how to make this fix.

Reply:
Tried to fix this problem putting generated files on the "target" folder. I couldn't fix this problem until now. Will be fixed later.

Reply:
I tried to omit the package name using a static import but was unsuccessful. Later, I can do this modification.